internal/core/domain/market.go

```diff
@@ -60,10 +63,10 @@ type Market struct {
 	BaseAssetPrecision uint
 	// Precison of the quote asset.
 	QuoteAssetPrecision uint
-	// Percentage fee expressed in basis points.
-	PercentageFee uint32
-	// Fixed fee amount expressed in satoshi for both assets.
-	FixedFee FixedFee
+	// Percentage fee expressed in basis points for both assets.
+	PercentageFee MarketFee
```

Comment:
Should we use same struct(MarketFee) for both PercentageFee(expressed in basis points) and FixedFee(expressed in sats)?

Reply:
Yes, that's what I did.